orb-slot-ctrl/src/lib.rs

```diff
@@ -174,6 +174,20 @@ impl TryFrom<u8> for RootFsStatus {
     }
 }
 
+impl TryFrom<&str> for RootFsStatus {
```

Comment:
The standard trait to use here is `FromStr` rather than `TryFrom`. 
You may then also be able to use this to provide a custom parser for the argument via clap.